pkgs/development/python-modules/asf-search/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytz, shapely, importlib-metadata, requests, python-dateutil }:
+
+buildPythonPackage rec {
+  pname = "asf_search";
+  version = "3.0.6";
+
+  src = fetchFromGitHub {
+    owner = "asfadmin";
+    repo = "Discovery-${pname}";
+    rev = "v${version}";
+    sha256 = "1jzah2l1db1p2mv59w9qf0x3a9hk6s5rzy0jnp2smsddvyxfwcyn";
+  };
+
+  doCheck = false;
```

Comment:
Can we execute the [tests](https://github.com/asfadmin/Discovery-asf_search/tree/master/tests)?

Reply:
Not for now. As far as I understand, we should wait for 3.1.0 for a working test workflow: https://github.com/asfadmin/Discovery-asf_search/blob/master/CHANGELOG.md